page/jquery-ui/widget-factory/how-to-use-the-widget-factory.md

```diff
@@ -299,6 +299,6 @@ $.widget( "custom.progressbar", {
 
 ## Closing Comments
 
-The widget factory is only one way of creating stateful plugins. There are a few different models that can be used and each have their own advantages and disadvantages. The widget factory solves lots of common problems for you and can greatly improve productivity, it also greatly improves code reuse, making it a great fit for jQuery UI as well as many other stateful plugins.
+The widget factory is only one way of creating stateful plugins. There are a few different models that can be used and each has their own advantages and disadvantages. The widget factory solves lots of common problems for you and can greatly improve productivity, it also greatly improves code reuse, making it a great fit for jQuery UI as well as many other stateful plugins.
```

Comment:
@ericcarraway shall we go all the way and change "each has their own" to "each has its own"?